tests/scripts/thread-cert/base_v1_2_LowPower_6_1_07_PreferringARouterOverAReed.py

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2020, The OpenThread Authors.
+#  All rights reserved.
+#
+#  Redistribution and use in source and binary forms, with or without
+#  modification, are permitted provided that the following conditions are met:
+#  1. Redistributions of source code must retain the above copyright
+#     notice, this list of conditions and the following disclaimer.
+#  2. Redistributions in binary form must reproduce the above copyright
+#     notice, this list of conditions and the following disclaimer in the
+#     documentation and/or other materials provided with the distribution.
+#  3. Neither the name of the copyright holder nor the
+#     names of its contributors may be used to endorse or promote products
+#     derived from this software without specific prior written permission.
+#
+#  THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+#  AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+#  IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+#  ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE
+#  LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR
+#  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+#  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+#  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+#  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+#  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+#  POSSIBILITY OF SUCH DAMAGE.
+#
+
+from mle import TlvType
+from pktverify import consts
+from pktverify.packet_verifier import PacketVerifier
+
+LEADER = 1
+ROUTER_1 = 2
+REED_1 = 3
+DUT = 4
+
+
+class LowPower_6_1_07_PreferringARouterOverAReed():
```

Comment:
```suggestion
class LowPower_6_1_07_PreferringARouterOverAReed(thread_cert.TestCase):
```

The current solution to skip running test for the base class works. But it has one issue that the IDE (e.g. PyCharm) can not autocomplete for the base class because it does not know that it's a subclass of `thread_cert.TestCase`. 
So, should we try other methods, e.g. https://stackoverflow.com/a/50922971 ?
Personally I think IDE autocomplete is important.

Reply:
The approach in https://stackoverflow.com/a/50922971  seems not work. I tried and noted the comment in this thread:

> This solution does not work with unittest's own test discovery/test runner. (I believe it requires using an alternate test runner, like nose.) – medmunds Jul 26 at 20:37

Shall we use another approach? Like [this](https://stackoverflow.com/questions/1323455/python-unit-test-with-base-and-sub-class/25695512#25695512)

Reply:
I think https://stackoverflow.com/a/25695512 is over complicated. 

Can you help try https://stackoverflow.com/a/22836015 and https://stackoverflow.com/a/17696807. 

I think https://stackoverflow.com/a/22836015 is very simple to use if it works.

Reply:
Decide to use `del` after discussion.

Reply:
@FiveDimensions @Irving-cl I think this can be the common practice for building test case where one DUT device has multiple roles.

Reply:
SGTM.